server/Makefile

```diff
@@ -153,6 +153,7 @@ PLUGIN_PACKAGES += mattermost-plugin-nps-v1.3.3
 PLUGIN_PACKAGES += mattermost-plugin-servicenow-v2.3.4
 PLUGIN_PACKAGES += mattermost-plugin-zoom-v1.8.0
 PLUGIN_PACKAGES += mattermost-plugin-ai-v0.6.3
+PLUGIN_PACKAGES += mattermost-plugin-focalboard-v8.0.0
```

Comment:
Let's split this change out to a separate PR that will prepackage boards given that is not related directly to the rest of the changes here.

Reply:
Sure.